core/templates/base.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>$Title$</title>
+</head>
+<body>
+$END$
```

Comment:
Could you remove this?

Reply:
Ok I'll look at google and get it taken care of